chingu-coders/Voyage2-Turtles-02
# user greeting prototype

## Purpose
* To allow user to specify a userName

## Approach/Fixes
* Implements a peripheral, but cool feature

## Testing
* [Check out this pull request](https://help.github.com/articles/checking-out-pull-requests-locally/)
* Confirm that on first load the greeting reads "Hello. What is your name?"
* Click on "name" and confirm that greeting disappears and text input appears
* Enter text to the input, press "enter" and confirm that userName is set
* Click on userName in greeting, and confirm that new userName can be input
* Close out browser, reload webpage, and confirm that userName persists
* [Open Chrome Developer Tools](https://developer.chrome.com/devtools), navigate to `application > Local Storage > http://localhost:3000` and confirm that a local storage item `userName` exists
* Still in Chrome Dev Tools Local Storage, Click on `userName` and press `delete`, reload the browser and confirm that no userName is set
